pset-iterators.js

```diff
@@ -137,11 +160,16 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter((presidentsAlive) => {
+  return presidentsAlive.death_year === null;
+})
 
+// Calling the callback livingPresidents in the console.
+console.log(livingPresidents)
```

Comment:
@OcielC Good job here on this solution using the filter method to return the presidents who are currently living.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)